Library/Formula/agda.rb

```diff
@@ -20,12 +21,33 @@ class Agda < Formula
 
   head "https://github.com/agda/agda.git", :branch => "master"
 
-  option "without-epic-backend", "Exclude the 'epic' compiler backend"
+  option "without-epic-backend", "Exclude the Epic compiler backend"
+  option "without-stdlib", "Don't install the Agda standard library"
+  option "with-malonzo-ffi", "Include the MAlonzo backend's FFI (depends on the standard library)"
 
   depends_on "cabal-install" => :build
   depends_on "ghc"
   depends_on "gmp"
   depends_on "bdw-gc" if build.with? "epic-backend"
+  depends_on "emacs" => :optional
+
+  head do
+    resource "stdlib" do
+      url "https://github.com/agda/agda-stdlib.git", :branch => "master"
+    end
+  end
+  devel do
+    resource "stdlib" do
+      url "https://github.com/agda/agda-stdlib/archive/v0.9.tar.gz"
+      sha1 "f39d5685ab2dc47758c87d9068047fce6b4b99a1"
+    end
+  end
+  stable do
+    resource "stdlib" do
+      url "https://github.com/agda/agda-stdlib/archive/v0.9.tar.gz"
+      sha1 "f39d5685ab2dc47758c87d9068047fce6b4b99a1"
+    end
+  end
```

Comment:
since devel and stable shared this resource definition, it can be placed outside of the blocks. But the head resource should stay in a block, preferrably combined with the `head` URL above.